extensions/Scratch_craft_2/modals.js

```diff
@@ -0,0 +1,72 @@
+// Name: Modals
+// ID: htmlalert
+// Description: This extension adds modal dialog boxes.
+// By: scratch_craft_2 <https://scratch.mit.edu/users/scratch_craft_2/>
+// License: MPL-2.0
+
+(function (Scratch) {
+    "use strict";
+
+  if (!Scratch.extensions.unsandboxed) {
+    throw new Error("This Extension must run unsandboxed");
+  }
+
+    class modals {
+      getInfo() {
+        return {
+          id: "htmlalert",
+          name: "modals",
```

Comment:
Extension title and all block text should contain Scratch.translate()
```suggestion
          name: Scratch.translate("Modals"),
```